src/drakvuf.cpp

```diff
@@ -125,13 +125,13 @@ static gpointer timer(gpointer data)
     return NULL;
 }
 
-int drakvuf_c::start_plugins(const bool* plugin_list, const char *dump_folder)
+int drakvuf_c::start_plugins(const char *dump_folder,
+                             const char *proctracer_config)
```

Comment:
Don't remove the bool* plugin_list input here. It was probably added after you branched off originally.